feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/widget/SessionToolbarBehavior.kt

```diff
@@ -0,0 +1,131 @@
+package io.github.droidkaigi.confsched2019.session.ui.widget
+
+import android.content.Context
+import android.util.AttributeSet
+import android.util.TypedValue
+import android.view.View
+import android.view.animation.AccelerateDecelerateInterpolator
+import android.widget.LinearLayout
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.appcompat.widget.Toolbar
+import androidx.coordinatorlayout.widget.CoordinatorLayout
+import androidx.core.view.ViewCompat
+import androidx.core.view.ViewPropertyAnimatorListener
+import androidx.core.view.children
+import androidx.core.widget.NestedScrollView
+import io.github.droidkaigi.confsched2019.session.R
+
+class SessionToolbarBehavior(
+    val context: Context? = null,
+    attrs: AttributeSet? = null
```

Comment:
I think `attrs: AttributeSet ` is not needed.

Reply:
Let me add information, Context and AttributeSet are required if this behavior is inflated from XML like `app:layout_behavior="io.github.droidkaigi.confsched2019.session.ui.widget.SessionToolbarBehavior"`
In other words, they are needless basically if we set this behavior via the code.

Reply:
Then, we can write this simply:

`SessionToolbarBehavior(): CoordinatorLayout.Behavior<LinearLayout>()`

Reply:
Thank you for your advices.
I want to use context.resources.getDimension() in the behavior class, then I rewrote like below

 ```
class SessionToolbarBehavior(
    val context: Context
) : CoordinatorLayout.Behavior<LinearLayout>(context, null)
```